test/TypeRoundTrip/Inputs/testcases/structural_types.swift

```diff
@@ -24,9 +24,7 @@ public func test() {
   roundTripType(Array<@convention(c) () -> ()>.self)
 
   // @convention(block) requires Objective-C support
-  #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
```

Comment:
You can use `#if _runtime(_ObjC)` to check for this instead.

Reply:
Indeed. It's still the wrong thing, though. This should work regardless of ObjC interop.